subsys/net/lib/sockets/sockets_tls.c

```diff
@@ -3028,11 +3028,18 @@ static int ztls_socket_data_check(struct tls_context *ctx)
 		/* MbedTLS API documentation requires session to
 		 * be reset in other error cases
 		 */
-		ret = tls_mbedtls_reset(ctx);
-		if (ret != 0) {
+		int err = tls_mbedtls_reset(ctx);
```

Comment:
Since we don't touch the error code, I'd just call `if (tls_mbedtls_reset(ctx) != 0) {` instead of introducing new variable.